src/frontend-samples/clashdetection-sample/EmphasizeElements.tsx

```diff
@@ -0,0 +1,72 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
This file is unnecessary.  Please replace callers with simple API calls like this:
    const emph = EmphasizeElements.getOrCreate(vp);
    emph.clearEmphasizedElements(vp);

Reply:
Deleted